app/scripts/templates/settings/clients.mustache

```diff
@@ -26,7 +26,8 @@
             {{^isCurrentDevice}}
             <div class="last-connected">{{lastAccessTimeFormatted}}</div>
             {{/isCurrentDevice}}
-            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect…{{/t}}</button>
+            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">
+            {{#isDevice}}{{#t}}Disconnect…{{/t}}{{/isDevice}}{{^isDevice}}{{#t}}Disconnect{{/t}}{{/isDevice}}</button>
```

Comment:
please see the style above in this template:

```html
             {{#isCurrentDevice}}
             <div class="device-current">{{#t}}current device{{/t}}</div>
             {{/isCurrentDevice}}
              {{^isCurrentDevice}}
              <div class="last-connected">{{lastAccessTimeFormatted}}</div>
              {{/isCurrentDevice}}
```

Let's apply the same to `isDevice` so it looks cleaner